content/_code-samples/build-a-wallet/desktop-js/library/4_helpers.js

```diff
@@ -0,0 +1,27 @@
+const xrpl = require("xrpl");
+
+const prepareTxData = (transactions) => {
+    return transactions.map(transaction => ({
+        confirmed: transaction.tx.date,
+        type: transaction.tx.TransactionType,
+        from: transaction.tx.Account,
+        to: transaction.tx.Destination,
```

Comment:
Since this function can get passed any type of transaction, we need to handle transactions that don't have a `Destination` field. This will return the string `-` if the field is undefined:

```suggestion
        to: transaction.tx.Destination ?? "-",
```